spring-xd-deployment.tex

```diff
@@ -37,13 +25,6 @@ \subsection{Lambda Architecture}
 on a single paradigm, the Lambda architecture employs both, allowing them
 to complement each other.
 
-\begin{figure}[ht]
-\centering
-\epsfig{file=lambda.eps, height=2in, width=3in}
-\caption{Lambda Architecture.}
-\label{fig:lambda}
-\end{figure}
-
 In the following subsections we will examine how the different components
 of the Lambda architecture can be implemented with Spring XD.
```

Comment:
I added some thoughts to the DEBS feedback document for this section:

https://docs.google.com/document/d/10wtQ_Gqcw_y0GxCZVe1ARJYlQ3M_ab6ZipUBFdr2GSY/edit#bookmark=id.ivfohl6w35zq